api/client/swarm/opts.go

```diff
@@ -30,9 +30,11 @@ const (
 
 var (
 	defaultPolicies = []swarm.Policy{
-		{Role: WORKER, Autoaccept: true},
-		{Role: MANAGER, Autoaccept: false},
+		{Role: worker, Autoaccept: true},
+		{Role: manager, Autoaccept: false},
 	}
+
+	exists = struct{}{}
```

Comment:
@lk4d4 Check out the comments on outdated parts of the diff in this PR.

> On Jul 11, 2016, at 09:44, Alexander Morozov <notifications@github.com> wrote:
> 
> In api/client/swarm/opts.go:
> 
> >  	}
> > +
> > +	exists = struct{}{}
> I wonder what the point of using struct{} instead of bool. bool syntax is much simpler.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>